app/src/main/java/com/example/navermovie/MainActivity.kt

```diff
@@ -1,85 +1,64 @@
 package com.example.navermovie
 
 import android.content.Context
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.telecom.Call
+import android.util.Log
 import android.view.inputmethod.InputMethodManager
 import android.widget.LinearLayout
+import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.LinearLayoutManager
-import com.google.gson.GsonBuilder
 import kotlinx.android.synthetic.main.activity_main.*
-import okhttp3.*
-import java.io.IOException
-import java.net.URL
-import java.net.URLEncoder
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
 
 class MainActivity : AppCompatActivity() {
 
-    val clientId = "kHLvopePGjYBNBI1P4Us"
-    val clientSecret = "GWjKNlXW_n"
+    private val clientId = "kHLvopePGjYBNBI1P4Us"
+    private val clientSecret = "GWjKNlXW_n"
```

Comment:
고정적인 값은 상수로 사용하는게 좋아요
```suggestion
    private const val CLIENT_SECRET = "GWjKNlXW_n"
```

Reply:
수정 완료했습니다!